integration-tests/kubernetes/maven-invoker-way/src/it/kubernetes-docker-build-and-deploy/invoker.properties

```diff
@@ -1,5 +1,5 @@
 # invoker.goals=clean package -Dquarkus.container.build=true -Dquarkus.package.type=native 
 # ensure that an attempt to deploy is made, but that the attempt fails (as we don't want to deploy this test application to a cluster that the runner of test may have configured)
-invoker.goals=clean package -Dquarkus.kubernetes.deploy=true -Dquarkus.kubernetes-client.master-url=http://localhost:12345
+invoker.goals=clean package -Dquarkus.kubernetes.deploy=true
 # expect a failure since there is no Kubernetes cluster to deploy to
-invoker.buildResult = failure
+invoker.buildResult = success
```

Comment:
Standard CI is broken:
```
2021-07-12T14:16:09.0349280Z [INFO] [ERROR] Failed to execute goal io.quarkus:quarkus-maven-plugin:999-SNAPSHOT:build (default) on project openshift-s2i-build-and-deploy: Failed to build quarkus application: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
2021-07-12T14:16:09.0385423Z [INFO] [ERROR] 	[error]: Build step io.quarkus.kubernetes.deployment.KubernetesDeployer#deploy threw an exception: java.lang.RuntimeException: Although a Kubernetes deployment was requested, it however cannot take place because there was an error during communication with the API Server at 'https://kubernetes.default.svc/'
2021-07-12T14:16:09.0395147Z [INFO] [ERROR] 	at io.quarkus.kubernetes.deployment.KubernetesDeploy.doCheck(KubernetesDeploy.java:90)
2021-07-12T14:16:09.0403816Z [INFO] [ERROR] 	at io.quarkus.kubernetes.deployment.KubernetesDeploy.check(KubernetesDeploy.java:36)
2021-07-12T14:16:09.0413392Z [INFO] [ERROR] 	at io.quarkus.kubernetes.deployment.KubernetesDeployer.deploy(KubernetesDeployer.java:84)
2021-07-12T14:16:09.0422398Z [INFO] [ERROR] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2021-07-12T14:16:09.0432481Z [INFO] [ERROR] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
2021-07-12T14:16:09.0472822Z [INFO] [ERROR] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2021-07-12T14:16:09.0480667Z [INFO] [ERROR] 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
2021-07-12T14:16:09.0489070Z [INFO] [ERROR] 	at io.quarkus.deployment.ExtensionLoader$2.execute(ExtensionLoader.java:820)
2021-07-12T14:16:09.0496283Z [INFO] [ERROR] 	at io.quarkus.builder.BuildContext.run(BuildContext.java:277)
2021-07-12T14:16:09.0503884Z [INFO] [ERROR] 	at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
2021-07-12T14:16:09.0512364Z [INFO] [ERROR] 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2442)
2021-07-12T14:16:09.0520910Z [INFO] [ERROR] 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1476)
2021-07-12T14:16:09.0528157Z [INFO] [ERROR] 	at java.base/java.lang.Thread.run(Thread.java:829)
2021-07-12T14:16:09.0534276Z [INFO] [ERROR] 	at org.jboss.threads.JBossThread.run(JBossThread.java:501)
2021-07-12T14:16:09.0543259Z [INFO] [ERROR] Caused by: io.fabric8.kubernetes.client.KubernetesClientException: An error has occurred.
2021-07-12T14:16:09.0553488Z [INFO] [ERROR] 	at io.fabric8.kubernetes.client.KubernetesClientException.launderThrowable(KubernetesClientException.java:64)
2021-07-12T14:16:09.0564360Z [INFO] [ERROR] 	at io.fabric8.kubernetes.client.KubernetesClientException.launderThrowable(KubernetesClientException.java:53)
2021-07-12T14:16:09.0633118Z [INFO] [ERROR] 	at io.fabric8.kubernetes.client.dsl.internal.ClusterOperationsImpl.fetchVersion(ClusterOperationsImpl.java:57)
2021-07-12T14:16:09.0644021Z [INFO] [ERROR] 	at io.fabric8.kubernetes.client.DefaultKubernetesClient.getVersion(DefaultKubernetesClient.java:501)
2021-07-12T14:16:09.0655236Z [INFO] [ERROR] 	at io.fabric8.kubernetes.client.DefaultKubernetesClient.getVersion(DefaultKubernetesClient.java:496)
2021-07-12T14:16:09.0664794Z [INFO] [ERROR] 	at io.quarkus.kubernetes.deployment.KubernetesDeploy.doCheck(KubernetesDeploy.java:71)
2021-07-12T14:16:09.0669233Z [INFO] [ERROR] 	... 13 more
2021-07-12T14:16:09.0675872Z [INFO] [ERROR] Caused by: java.net.UnknownHostException: kubernetes.default.svc
2021-07-12T14:16:09.0683516Z [INFO] [ERROR] 	at java.base/java.net.InetAddress$CachedAddresses.get(InetAddress.java:797)
2021-07-12T14:16:09.0691177Z [INFO] [ERROR] 	at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1509)
2021-07-12T14:16:09.0698807Z [INFO] [ERROR] 	at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1368)
2021-07-12T14:16:09.0706201Z [INFO] [ERROR] 	at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1302)
2021-07-12T14:16:09.0712152Z [INFO] [ERROR] 	at okhttp3.Dns.lambda$static$0(Dns.java:39)
```
I would bet it's related to these changes.